src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,11 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={todo.completed ? `TodoInfo TodoInfo--completed` : 'TodoInfo'}>
```

Comment:
it's better (looks more laconic) to use libraries as `classnames` in such cases

Reply:
What are you mean `libraries as classnames in such cases'?